third_party/amd/backend/driver.py

```diff
@@ -37,7 +37,7 @@ class DlPhdrInfo(ctypes.Structure):
     # Load libc and get the dl_iterate_phdr symbol.
     try:
         dl_iterate_phdr = ctypes.CDLL('libc.so.6').dl_iterate_phdr
-    except:
+    except Exception:
```

Comment:
Almost the same thing (in this context it seems that such a replacement is correct, only a small number of system exceptions will not be caught, like `SystemExit` and so on), but ruff doesn't complain.